components/pthread/pthread.c

```diff
@@ -221,6 +245,28 @@ int pthread_create(pthread_t *thread, const pthread_attr_t *attr,
         if (pthread_cfg->prio && pthread_cfg->prio < configMAX_PRIORITIES) {
             prio = pthread_cfg->prio;
         }
+
+        if(pthread_cfg->inherit_cfg) {
+            if(pthread_cfg->thread_name == NULL) {
+                // Inherit task name from current task.
+                task_name = pcTaskGetTaskName(NULL);
+            }
+            else {
```

Comment:
Nitpick, `else` should be on the same line with the closing brace; also there is a missing space between `if` and opening paren. Same in a few other places below.

Reply:
Hm, guess I did I forget to run the formatter. Will fix.